views/products/demo/drynx.tpl

```diff
@@ -27,9 +27,9 @@
 <img src="../../../resources/products/images/drynx/demo-year-claim.png" width="60%" class="center"/>
 
 <p>You can see that older houses tend to have higher claims than newer houses, which seems
-intuitively correct. To test it out yourself, you can visit the
-<a href="https://factory.c4dt.org/incubator/drynx/demo">Drynx Demonstrator</a></p>
+intuitively correct.</p>
 
-<a href="https://factory.c4dt.org/incubator/drynx/demo" class="tab-link">
-    For a Live Demo Click Here
-</a>
+<p>
+    We do not host this demo anymore but you can
+    <a href="https://github.com/c4dt/service-drynx">find its code here</a>.
```

Comment:
The `service-drink` needs to have a README with the following information:
- link to the showcase
- link to the paper
- explanation how to run the demo
- 1 or 2 screenshots

Reply:
I followed the README and have a local installation (which throws some errors, but I'm not familiar enough with the code to know if they are "normal"), and there's a link to the showcase as well as a screenshot, but no link to the paper.

Reply:
Correct - added it to the https://github.com/c4dt/TODO/issues/345